src/jit/hwintrinsicxarch.cpp

```diff
@@ -124,7 +124,7 @@ InstructionSet Compiler::lookupHWIntrinsicISA(const char* className)
 //    isa        -- instruction set of the intrinsic.
 //
 // Return Value:
-//    Id for the hardware intrinsic.
+//    Id for the hardware intrinsic
```

Comment:
Not sure why this change was made. Although many comments lack a '.' the function guidelines illustrate their use in this context.

Reply:
This is result of rolling back previous changes to return to original state, what seems was not as precise as it should be.